src/RestClient/endpoints/V3/Themes/ThemeConfigurations.ts

```diff
@@ -0,0 +1,37 @@
+import { AxiosInstance } from 'axios';
+
+import { themesPath } from './Themes';
+import type { ThemeConfiguration } from './types';
+
+class ThemeConfigurations {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Returns a list of theme configurations
+   *
+   * @param uuid A valid Theme UUID
+   * @param params Query parameters used to filter response
+   * @returns Promise resolving to a list of theme configurations
+   */
+  list(uuid: string, params?: ThemeConfiguration['ListFilters']): ThemeConfiguration['ListResponse'] {
+    return this.client.get(`${themesPath}/${uuid}/configurations`, {
+      params,
+    });
+  }
+
+  /**
+   * Validates a Theme Configuration
```

Comment:
I would recommend changing this description to be more detailed and descriptive of what this method is used for. Our docs have a pretty good description: 
"Validates a theme configuration against the theme's schema without creating it. Useful for testing schemas before creation."